wai-middleware-throttle.cabal

```
@@ -34,7 +34,7 @@ library
                      , hashable              >= 1.2
                      , http-types
                      , mtl
-                     , network               >= 2.1
+                     , network               >= 2.1 && <2.9
```

While you can't easily verify this with Stackage (even though network-3.0 was released half a year ago, to this day there still isn't any Stackage snapshot featuring with a version >= 3.0) you did merge the PR which made `wai-middleware-throttle` compatible with network-3.[01], thus the upper bound can be set higher, i.e.

```suggestion
                     , network               >= 2.4.2 && <3.2
```

I'd also set the lower bound higher as I ran into build-failures with older versions (e.g. due to a missing `Ord` instance on `Address` which wai-middleware-throttle relies on; this is also something you can't easily verify with Stackage snapshots as there's no Snapshot available which has a low enough version of `network` in order to verify the lower bound)